### DISCUSSION TITLE
Change logo image on dark/light revealjs slides

### DISCUSSION THREAD

**Discussion by jthomasmock** [quarto-dev/quarto-cli#3417] [Feature Requests]
For revealjs slides there are often times when the slide color shifts from light to dark, for sections breaks, summaries, etcs.

It would be nice to be able to identify a dark/light logo that is then used uses the same logic as the font-color choice (revealjs auto chooses light text on dark and dark on light).

**Comment by jthomasmock** [comment]
I wasn't able to get the workaround accomplished in revealjs as the logo via YAML is a bit trickier: https://github.com/quarto-dev/quarto-cli/discussions/1647

**Reply by mcanouil** [reply]
FYI #5439 (I'll try with `revealjs` to replicate).

**Comment by sagikazarmark** [comment]
Incidentally, this has been working until #4031 got "fixed".

Here is the workaround I use for now to disable logos on dark backgrounds:

```scss
.reveal.has-dark-background .slide-logo {
  display: none !important;
}
```

I could imagine a data- attribute that specifically allows disabling the logo on certain slides.

In addition, it would also be nice if the logo followed the background transition settings. Right now, it just appears/disappears (when the above hack is used).

**Comment by mcanouil** [comment]
See https://github.com/quarto-dev/quarto-cli/discussions/5439#discussioncomment-5828141 for code.
It will work as long as you have a "-light" and a "-dark" version of the images.

https://user-images.githubusercontent.com/8896044/236789870-279d508e-25d6-41b5-baac-f4823d596048.mov
